### ISSUE TITLE
NuGet package version 3.2.0 is not properly signed

### ISSUE THREAD

**Issue by timvandenhof** [Azure/azure-notificationhubs-dotnet#92]
NuGet package version 3.2.0 seems to be delay signed and not fully signed as expected. Running our application with the new package results in the error below. Running sn.exe reveals that the assembly is delay signed. Running the same verification on version 3.1.0 results in a valid assembly. Seems to be 3.2.0 specific.

```
Could not load file or assembly 'Microsoft.Azure.NotificationHubs' or one of its dependencies. Strong name signature could not be verified.  The assembly may have been tampered with, or it was delay signed but not fully signed with the correct private key. (Exception from HRESULT: 0x80131045) 
```

```
sn -vf Microsoft.Azure.NotificationHubs.dll

Microsoft (R) .NET Framework Strong Name Utility  Version 4.0.30319.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Microsoft.Azure.NotificationHubs.dll is a delay-signed or test-signed assembly
```

**Comment by marstr** [comment]
Howdy @timvandenhof,

We've temporarily unlisted 3.2.0 to prevent further downloads of the delay-signed bits, and are working to resolve this issue now. Sorry for the inconvenience!

**Comment by mpodwysocki** [comment]
@timvandenhof Fixed as per [3.2.1](https://github.com/Azure/azure-notificationhubs-dotnet/releases/tag/3.2.1).  Thank you for reporting the issue!
